_pkgdown.yml

```diff
@@ -25,12 +25,15 @@ navbar:
 
 articles:
   - title: "Getting Started"
+    navbar: Get started
     contents:
       - getting-started-with-teal-reporter
   - title: "Core Concepts"
+    navbar: Get started
     contents:
       - teal-report-class
   - title: "Building Reportable App"
+    navbar: Get started
     contents:
       - reportable-shiny-application
```

Comment:
Not sure about having repeating "Get Started" section.

<img width="326" height="313" alt="image" src="https://github.com/user-attachments/assets/7e9d144e-f1a9-40b9-aa35-dab80d032c72" />

How about just remove the `articles` modification so it will show like this:

<img width="337" height="163" alt="image" src="https://github.com/user-attachments/assets/116de2ed-ad79-48c3-89b3-69b29e8f8838" />

All I did was delete these `articles` section lines:
https://github.com/insightsengineering/teal.reporter/blob/main/_pkgdown.yml#L26-L35

Reply:
Done

<img width="875" height="210" alt="image" src="https://github.com/user-attachments/assets/b32b75b6-4552-4b28-aac5-b94d4b2d0b71" />